src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -265,8 +265,9 @@ def load_arguments(self, _):
         VMPriorityTypes = self.get_models('VirtualMachinePriorityTypes', resource_type=ResourceType.MGMT_COMPUTE)
         c.argument('name', name_arg_type)
         c.argument('nat_backend_port', default=None, help='Backend port to open with NAT rules.  Defaults to 22 on Linux and 3389 on Windows.')
-        c.argument('single_placement_group', default=None, help="Enable replicate using fault domains within the same cluster", arg_type=get_enum_type(['true', 'false']))
-        c.argument('platform_fault_domain_count', default=None, type=int, help='Fault Domain count for each placement group in the availability zone', min_api='2017-12-01')
+        c.argument('single_placement_group', arg_type=get_three_state_flag(), help="Enable replicate using fault domains within the same cluster. Default to 'false' for any zonals, or with 100+ instances, or Fault Domain count set at 1"
+                   " See https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-placement-groups for details")
```

Comment:
This should really be in the long-summary for this param, so this would need to be in the `help.py` file.